docs/metrics/perplexity.md

```diff
@@ -0,0 +1,148 @@
+# Perplexity
+
+!!! info inline end "Requirements"
+
+    It is important to note that perplexity can only be calculated using [autoregressive language models](#autoregressive-language-model-info).
+
+Perplexity is a metric commonly used in natural language generation to evaluate the quality of language models,
+particularly in the context of text generation. Unlike metrics such as [BLEU](bleu.md) or [BERT](bertscore.md),
```

Comment:
Does it make sense to say in NLG particularly in the context of text generation? I feel like it's redundant now.

Reply:
Agreed, I've changed it back to NLP. I think it flows better than NLG